mods/addScore.js

```diff
@@ -6,7 +6,9 @@ const newScore = (user, score) => {
   const player = document.createElement('li');
   player.innerHTML = `
                     <div class="playerDetails">
-                     <p class="playerName">${user}:</p>
+                     <p class="playerName">
+                     ${user}:
+                     </p>
                      <p class="playerScore">${score}</p>
                     </div>
                    `;
```

Comment:
- [ ] **[OPTIONAL]** Your app does not really work as expected. After adding my name and score, when I refresh nothing it does not appear.

  I put in my name and score | Refresh button is pressed (name does not appear)
  :---:|:---:
  ![natasha-1](https://user-images.githubusercontent.com/36542986/181743000-4f8995df-5ffe-4957-97c8-d3f590266ae3.jpg) | ![natasha-2](https://user-images.githubusercontent.com/36542986/181742988-76a75c6d-169d-4866-96ff-ed853e1010cc.jpg)